api/module.mk

```diff
@@ -3,20 +3,23 @@
 REQUIRED_2P_REPOSITORIES := pipeline-config warp
 include $(MAKE_INCLUDE_DIR)/modules.mk
 
+CLASSES_DIR           := $(MODULE_DIR)/classes
 CPCACHE_DIR           := $(MODULE_DIR)/.cpcache
 RESOURCES_DIR         := $(MODULE_DIR)/resources
 SRC_DIR               := $(MODULE_DIR)/src
 TEST_DIR              := $(MODULE_DIR)/test
 TEST_RESOURCES_DIR    := $(TEST_DIR)/resources
 DERIVED_RESOURCES_DIR := $(DERIVED_MODULE_DIR)/resources
-DERIVED_SRC_DIR		  := $(DERIVED_MODULE_DIR)/src
+DERIVED_SRC_DIR       := $(DERIVED_MODULE_DIR)/src
 DERIVED_TARGET_DIR    := $(DERIVED_MODULE_DIR)/target
-BOOT_PROJECT          := $(MODULE_DIR)/build.boot
 CLOJURE_PROJECT       := $(MODULE_DIR)/deps.edn
-LEIN_PROJECT          := $(MODULE_DIR)/project.clj
 
-CLEAN_DIRS  += $(CPCACHE_DIR)
-CLEAN_FILES += $(LEIN_PROJECT) $(MODULE_DIR)/wfl
+API_DIR      := $(CLASSES_DIR)/wfl/api
+POM_IN       := $(MODULE_DIR)/pom.xml
+POM_OUT      := $(SRC_DIR)/META-INF/maven/org.broadinstitute/wfl/pom.xml
```

Comment:
If these are generated, is it possible to output them into `dervied/api`?

Reply:
Evidently, not yet -- at least, not POM_IN from the command line.

https://github.com/clojure/tools.deps.alpha/blob/c971a662730e75ee05186a20a776bb37390ee127/src/main/clojure/clojure/tools/cli/api.clj#L85

Looking into POM_OUT now, and it should be possible to load deps as a library.